src/renderers/dom/shared/DOMProperty.js

```diff
@@ -13,10 +13,26 @@
 
 var invariant = require('fbjs/lib/invariant');
 
+// These attributes should be all lowercase to allow for
+// case insensitive checks
+var RESERVED_PROPS = {
+  children: true,
+  dangerouslysetinnerhtml: true,
+  autofocus: true,
+  defaultvalue: true,
+  defaultchecked: true,
+  innerhtml: true,
+  suppresscontenteditablewarning: true,
+  onfocusin: true,
+  onfocusout: true,
```

Comment:
Good catch. I these from `possibleStandardNames` to prevent the UnknownDOMPropertyHook from picking it up in addition to the dev warning about `onfocusin`.

I think we just need to make the `onFocusIn` warning case insensitive. What do you think?